web/src/main/bvmui/src/components/Card.vue

```diff
@@ -1,28 +1,44 @@
 <template>
-    <div class="bvm-card" @click="$emit('click', $event)">
-        <div class="bvm-card-description">{{data.description}}</div>
-        <div class="bvm-card-status">
-            <h1 >{{data.state}}</h1>
```

Comment:
this should be explicitly stated, using colours is not enough expecially for an operator that is the first time that uses the tool

Reply:
ok